ts/table-grid/table-grid.directive.ts

```diff
@@ -39,11 +39,14 @@ module Lui.Directives {
 				let bodies: any = tablegrid.querySelectorAll("tbody"); 			// Both table bodies
 
 				let lockedColumns: any = tablegrid.querySelector(".locked.columns");
+				let lockedColumnsVS: any = (!!lockedColumns) ? lockedColumns.querySelector(".holder .virtualscroll") : undefined;
 				let lockedColumnsSynced: any = lockedColumns ? lockedColumns.querySelector(".holder") : undefined;
 
 				let scrollableArea: any = tablegrid.querySelector(".scrollable.columns"); // scrollable area
 				let scrollableAreaVS: any = scrollableArea.querySelector(".virtualscroll");
 
+				let minRowsCountForVS = 200;
```

Comment:
when you define a magic number, could you use ALL_CAPS and comment its a magic number

see [here](https://github.com/LuccaSA/lucca-ui/blob/master/js/directives/lucca/user-picker.js#L11)